layers-samples/proj-multiview-occlusion.html

```diff
@@ -0,0 +1,303 @@
+<!doctype html>
```

Comment:
filename is a bit odd, since this looks like it will work without multiview, and simply uses multiview as an optimization when available?

Reply:
I don't think this sample will work without multiview.  
It seems there's an inline option but maybe I should remove it?